omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/ConceptResource1_9Test.java

```diff
@@ -34,6 +27,18 @@
 import org.openmrs.module.webservices.rest.web.response.ConversionException;
 import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_8.ConceptResource1_8;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+import static junit.framework.TestCase.assertFalse;
+import static junit.framework.TestCase.assertTrue;
+import static org.hamcrest.CoreMatchers.*;

```

Comment:
Btw we should probably be forgiving of wildcard imports of test libraries
in test code.

(But no wildcard imports under src/main.)

-Darius (by phone)
On Jun 7, 2016 12:35 PM, "dkayiwa" <notifications@github.com> wrote:

> In
> omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/ConceptResource1_9Test.java
> <https://github.com/openmrs/openmrs-module-webservices.rest/pull/216#discussion_r66137511>
> :
>
> > @@ -34,6 +27,18 @@
> >  import org.openmrs.module.webservices.rest.web.response.ConversionException;
> >  import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_8.ConceptResource1_8;
> >
> > +import java.util.ArrayList;
> > +import java.util.List;
> > +import java.util.Locale;
> > +
> > +import static junit.framework.TestCase.assertFalse;
> > +import static junit.framework.TestCase.assertTrue;
> > +import static org.hamcrest.CoreMatchers.*;
>
> We do not encourage using wild card imports
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openmrs/openmrs-module-webservices.rest/pull/216/files/75c040f9c8cc0f8b3498eae7ba6429184399832e#r66137511>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABHHtKVDjFfkbvQ9b7ckjBQfe2B5GwTnks5qJcfkgaJpZM4IurrI>
> .
>